sqlserver/assets/configuration/spec.yaml

```diff
@@ -135,6 +135,28 @@ files:
       value:
         type: integer
         example: 5
+    - name: custom_queries
+      description: |
+        Get metrics from a custom SQL query but only if the database is writable
+        (i.e. it's the master in an availability group) Notes:
+        - Custom SQL query must be defined in its own instance
+        - For performance counter queries, use `custom_metrics` instead
+        See XXX for more information.
+      value:
+        type: array
+        items:
+          type: object
+          properties: []
+        example:
+          - metric_prefix: sqlserver
```

Comment:
Let's remove the prefix option as that was a legacy thing https://github.com/DataDog/integrations-core/blob/fe569452d79577b79b07884c0ba04d01e1cb2b51/oracle/datadog_checks/oracle/oracle.py#L65-L66

Reply:
As an alternative just always prepend `sqlserver.` to the metric name instead?